tests/sanity/test_multus_v4_0_2.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import subprocess
+import os
+
+from charmed_kubeflow_chisme.rock import CheckRock
+
+
+def ensure_image_contains_paths(image, paths):
+    for path in paths:
+        subprocess.run(
+            [
+                "docker", "run", "--rm", image,
+                "ls", "-l", path
```

Comment:
you can check all the paths at once, rather than check each file independently. Checking them simultaneously will shorten the execution time significantly, though if it fails, I don't think you'd know which path doesn't exist.

Additionally, this could be in a ``utils.py`` module, or in a common library, as we all need to do something like this.

Reply:
I went with the simplest approach. "ls" is included in bare images, while afaik "find" isn't. There's another potential issue with listing all files: I'm not sure how much output we can get before it gets trimmed.

Fwiw, it's being moved to the shared repo, so we can improve the implementation at any time: https://github.com/canonical/k8s-test-harness/pull/1/files#diff-014884dfb84180e06e68a13330efae19a3dc39a4dfb58f95db2992c3326512c9R9-R14.